packages/Button/src/Button.stories.tsx

```diff
@@ -179,17 +197,37 @@ export const Sizes = () => (
 );
 
 export const States = () => (
-  <Section>
-    <Button appearance="secondary">Secondary</Button>
-    <Button disabled appearance="secondary">
-      Secondary
-    </Button>
-    <Button active appearance="secondary">
-      Secondary
-    </Button>
-    <Button className="focus" appearance="secondary">
-      Secondary
-    </Button>
+  <Section column>
+    <Section>
+      <Button appearance="secondary">Secondary</Button>
+      <Button disabled appearance="secondary">
+        Secondary
+      </Button>
+      <Button active appearance="secondary">
+        Secondary
+      </Button>
+      <Button className="focus" appearance="secondary">
+        Secondary
+      </Button>
+    </Section>
+
+    <Section>
+      <Button appearance={{ type: 'ghost', variant: 'secondary' }}>
+        Secondary
+      </Button>
+      <Button disabled appearance={{ type: 'ghost', variant: 'secondary' }}>
+        Secondary
+      </Button>
+      <Button active appearance={{ type: 'ghost', variant: 'secondary' }}>
```

Comment:
I don't seem to have seen an active state for the Ghost Button in Figma, to be validated with the design team

Reply:
I validated and there should be an active state.